docs/content/docs/reference/hasura-auth/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Overview'
+---
+
+Hasura Auth handles **authentication** for [Hasura](https://github.com/hasura/graphql-engine).
+
+It runs in a separate Docker container along side Postgres and Hasura.
+
+## Features
+
+- 🧑‍🤝‍🧑 Users are stored in Postgres and accessed via GraphQL
+- 🔑 Multiple sign-in methods
+- ✨ Integrates with GraphQL and Hasura Permissions
+- 🔐 JWT tokens and Refresh Tokens.
+- ✉️ Emails sent on various operations
+<!-- - ✅ Optional checking for Pwned Passwords. -->
+- Two-factor authentication support.
```

Comment:
would be great if we could have some emoji on this line too, that would make this section more consistent

Reply:
🛡️ maybe? I'm not sure. I would use the lock with the key, but that's already reserved.